### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by Rana-Mudassir**
# In this PR following tasks are done: 

- [ ] Create a method called current_user in your ApplicationController to make current user data available to all controllers.
- [ ] It will return the first user from the database.
- [ ] Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- [ ] Create forms to perform the following functions:
- [ ] Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
- [ ] Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- [ ] Allow Users to add likes to Posts.

Thanks for Reviewing 😄

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <p><%= @post.text %></p>
+  </div>
+
+   <div class="comment">
+    <%= form_with(model: Comment.new, url: user_post_comments_path(@post.author, @post)) do |form| %>
+      <%= form.text_area :text, placeholder: 'Enter your comment', rows: 4 %>
+      <%= form.submit 'Add Comment' %>
+    <% end %>
+  </div>
+
```
Comment (Owusu-Desmond) [inline app/views/posts/show.html.erb]: - [ ] Please make sure that there is a separate form in the `views/comments/new.html.erb` file that accepts input for creating comments :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Rana-Mudassir , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (Rana-Mudassir) [reply app/views/posts/show.html.erb:19]: Hi @Owusu-Desmond  Will you please elaborate about this required Changes as this is not specified in the project requirement and I don't understood your required changes, will you show me some example about this. Thanks

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Rana-Mudassir   ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [ ] Good code organization and working db ✔️
- [ ] Nice use of Rails form structure ✔️ 
- [ ] Descriptive README ✔️


Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**